src/Link.php

```diff
@@ -15,9 +16,13 @@ public function __construct(
         private readonly string $label = '',
         private readonly string|null $url = null,
         private readonly bool $active = false,
+        private readonly bool $disabled = false,
         private readonly bool $encodeLabel = true,
         private readonly array $attributes = [],
     ) {
+        if ($this->active && $this->disabled) {
+            throw new InvalidArgumentException('The link cannot be both active and disabled.');
+        }
```

Comment:
I think we shouldn't add such constrains.

Reply:
A link cannot be in both an active and disabled state at the same time, so it is correct to throw an exception.

Reply:
Currently "disabled" property is not used and can be deleted.

Reply:
done.